pkg/operator/k8sutil/prometheus.go

```diff
@@ -1,3 +1,20 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
```

Comment:
2020 for the copyrights?

Reply:
I'm not sure if we can just update the year for the copyright as the file already existed. Though I can push it to around 2018/2019 as that should be when those files have been created.
What do you think?